### PR TITLE
#144187585 Display real time results

### PR DESCRIPTION
**Pull request by tezebuike**
**What does this PR do?**

This PR ensures that a realtime search result is displayed as user searches for keywords

**Description of Task to be completed?**

Display relevant results on the page as a user is typing rather than wait until they submit the search query

**How should this be manually tested?**

Start the application, click on the search icon and then type to see matching results live.

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

#144187585 Display real-time results as shopper is typing.

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+exclude_paths:
```
Comment (jomadoye) [inline .codeclimate.yml]: Please i don't think this file should be in this PR

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - if [ ! -e $HOME/.meteor/meteor ]; then curl https://install.meteor.com | sh; fi
+  - npm i -g reaction-cli coveralls
+  - meteor npm install
+
+script: 
+  - reaction test
+
+after_success:
+  - npm install -g codeclimate-test-reporter
+  - codeclimate-test-reporter < lcov.info
```
Comment (jomadoye) [inline .travis.yml]: Please i don't think this file should be in this PR

File: package.json [hunk 3]
```diff
@@ -103,6 +103,7 @@
     "babel-preset-stage-2": "^6.22.0",
     "browserstack-local": "^1.3.0",
     "chai": "^3.5.0",
+    "coveralls": "^2.13.1",
```
Comment (jomadoye) [inline package.json]: Please i don't think this file should be in this PR

**Review by jomadoye — DISMISSED** [review]

File: .codeclimate.yml (hunk 1 above)
Comment (tezebuike) [reply .codeclimate.yml]: Fixed

File: .travis.yml (hunk 2 above)
Comment (tezebuike) [reply .travis.yml]: Fixed

File: package.json (hunk 3 above)
Comment (tezebuike) [reply package.json]: Fixed

**Comment by fmustapha** [comment]
Good job!

**Review by fmustapha — APPROVED** [review]

**Review by tonymontaro — APPROVED** [review]
All good!

**Pull request merged**